src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('tr');
+const tableBody = document.querySelector('tbody');
+const toNumber = (string) => {
+  if (typeof string === 'string') {
+    const clearNum = string.replace(/[$, ]/g, '');
```

Comment:
The 'replace' regex is designed to remove dollar signs, commas, and spaces, but it doesn't account for other currency symbols or thousands separators used in different locales. Ensure this function handles the expected input formats.